modules/stickychan.cpp

```diff
@@ -191,6 +191,28 @@ class CStickyChan : public CModule
 		return false;
 	}
 
+	EModRet OnRaw(CString& sLine) override {
+		CString sNumeric = sLine.Token(1);
```

Comment:
The latest master requests server-time when available so we can no longer assume tokens. I suppose this would be fine in the 1.6.x branch, though.